### DISCUSSION TITLE
How to place a word index in the thesis template?

### DISCUSSION THREAD

**Discussion by Jmmo-Gomes** [joaomlourenco/novathesis#396] [Q&A] [answered]
How can I include a word index in the end of the template like the following?

![image](https://user-images.githubusercontent.com/72549192/206811664-5ffac165-4574-40ad-aec9-24941b6b4cde.png)

**Comment by joaomlourenco** [comment]
Hi @Jmmo-Gomes,

That's an excellent question.  It should be easy, but I never addressed this “problem”.  Let me investigate!

**Comment by joaomlourenco** [comment]
I will convert this to an issue, and future interactions will happen there.

**Comment by joaomlourenco** [comment, marked as answer]
@Jmmo-Gomes
A versão mais recente no repositório suporta um índice de palavras. É só descomentar a opção `printindex=true` no ficheiro "Config/1_novathesis.tex" e depois adicionar entradas com o comando `\index{palavra(s)}`.
